docs/stories/components/loadings/LLoading.stories.js

```diff
@@ -0,0 +1,20 @@
+import LLoading from '@components/loadings/LLoading.vue';
+
+export default {
+  title: 'Design System/Components/Base/Loading',
+  component: LLoading,
+  argTypes: {
+    size: { control: 'text', description: 'Loading icon size'},
```

Comment:
Só adicionar um espaço antes de fechar as chaves :)
```suggestion
    size: { control: 'text', description: 'Loading icon size' },
```